files/examples/tls-rate2-targetgrouppolicy.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: application-networking.k8s.aws/v1alpha1
+kind: TargetGroupPolicy
+metadata:
+    name: tls-rate2
+spec:
+    targetRef:
+        group: "application-networking.k8s.aws"
+        kind: ServiceExport
+        name: tls-rate2
+    protocol: TCP
+    healthCheck:
+         enabled: false
```

Comment:
do we want to include an example with health check enabled?

Reply:
The new example nginx image: `public.ecr.aws/x2j8p8w7/https-server:latest` has some issue to enable health check. the targets are healthy initially but it will then change to unhealthy. Still plan to keep the healthcheck disable as unhealthy target will be weighted out and traffic will not work.


But in our other TargetGroupPolicy, it did have healthcheck enabled examples: 

https://github.com/aws/aws-application-networking-k8s/blob/112b6b0cc3e79df166a410b8580786d65f2fb83c/docs/api-types/target-group-policy.md?plain=1#L36-L62